ui/src/features/Dashboard/components/Nav/components/UserNavigation.tsx

```diff
@@ -17,17 +18,19 @@ import { useDialogType } from "@flow/stores";
 const UserNavigation: React.FC = () => {
   const t = useT();
   const [, setDialogType] = useDialogType();
+  const { logout: handleLogout, user } = useAuth();
+
   return (
     <DropdownMenu>
       <DropdownMenuTrigger>
         <div className="flex gap-2 mr-2">
           <Avatar className="h-8 w-8">
-            <AvatarImage src="https://github.com/KaWaite.png" />
-            <AvatarFallback>KW</AvatarFallback>
+            <AvatarImage src={user?.picture ? user.picture : "https://github.com/KaWaite.png"} />
```

Comment:
As a fallback can you replace my thumbnail perhaps with a U for user (to keep inline with the avatar fallback and username)